src/common/dao/group/usergroup.go

```diff
@@ -92,3 +92,29 @@ func UpdateUserGroupName(id int, groupName string) error {
 	_, err := o.Raw(sql, groupName, id).Exec()
 	return err
 }
+
```

Comment:
We don't need another `group` pkg...  I recall it was introduced due to some dependency issue, and there should not be any dependencies here.  `dao` should NOT depends on `auth`